### PR TITLE
Fill out the README.md

### PR DESCRIPTION
**Pull request by imtayadeway**
Fixes https://github.com/ManageIQ/manageiq-api/issues/120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # ManageIQ::Api
-Short description and motivation.
+
+ManageIQ::Api is a plugin to be used in the [ManageIQ project][ManageIQ]. It forms
+the public JSON API for ManageIQ.
```
Comment (chrisarcand) [inline README.md]: s/JSON/REST/, not because it's inaccurate but because we refer to it as the REST API basically everywhere so why not here, too.

**Comment by chrisarcand** [comment]
LGTM 👍   👉 @abellotti

**Comment by abellotti** [comment]
Thanks @imtayadeway for this update.  

For me the CONTRIBUTING and LICENSE links both got me 404's, are these going to be added in follow-up PRs or are the links bad ? Thanks.

**Comment by imtayadeway** [comment]
@abellotti thanks for checking those! The license link was already there and broken, so I just copied that! They should both be fixed now 👍

**Comment by abellotti** [comment]
LGTM!! Thanks for fixing this @imtayadeway  🎵

**Pull request merged**